service/verifyService.go

```diff
@@ -0,0 +1,20 @@
+package service
+
+import (
+	"net/http"
+
+	"github.com/Real-Dev-Squad/discord-service/utils"
+	"github.com/bwmarrin/discordgo"
+)
+
+func (s *CommandService) VerifyService(response http.ResponseWriter, request *http.Request) {
```

Comment:
```suggestion
func (s *CommandService) Verify(response http.ResponseWriter, request *http.Request) {
```
do we need to explicitly declare method with suffix as service?

Reply:
done